internal/src/env.rs

```diff
@@ -9,7 +9,7 @@ pub const DYLINT_LIBRARY_PATH: &str = "DYLINT_LIBRARY_PATH";
 pub const DYLINT_LIBS: &str = "DYLINT_LIBS";
 pub const DYLINT_LIST: &str = "DYLINT_LIST";
 pub const DYLINT_RUSTFLAGS: &str = "DYLINT_RUSTFLAGS";
-pub const HOME: &str = "HOME";
+pub const HOME: &str = "HOME"; // Todo: Remove this on the next major semver change.
```

Comment:
```suggestion
```
This package is meant to be internal to Dylint. So I have no problem removing this line now.